lib/intercom/version.rb

```diff
@@ -1,3 +1,3 @@
 module Intercom #:nodoc:
-  VERSION = "3.9.3"
+  VERSION = "3.10.0"
```

Comment:
Do you mind if we leave this change and then bump in a separate PR? 

Modifying this version here affects the automatic bump command that we use when bundling releases 😄

Reply:
Of course!